resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -281,6 +298,10 @@ class VisualEffect
         {
             return $false
         }
+        if ($this.TransparencyEffectsSetting -ne $currentState.TransparencyEffectsSetting)
+		{
+            return $false
+        }

```

Comment:
```suggestion
        if (($null -ne $this.TransparencyEffects) -and ($this.TransparencyEffects -ne $currentState.TransparencyEffects))
	{
            return $false
        }

```

Need to first check if $this.TransparencyEffects is $null now that the parameter is nullable. Be sure to also do the same for $this.AlwaysShowScrollbars